scheduler/src/cook/cached_queries.clj

```diff
@@ -20,19 +20,25 @@
   [job-ent]
   (caches/lookup-cache-datomic-entity! caches/job-ent->user-cache :job/user job-ent))
 
-(defn instance-uuid->job-uuid-datomic-query
+(defn pod-name->job-ent-datomic-query
```

Comment:
Maybe this should be `instance-uuid->job-ent-datomic-query` since we're still assuming the input is a valid instance UUID?